decidim-core/db/migrate/20170119150649_add_show_statistics_to_organization.rb

```diff
@@ -0,0 +1,5 @@
+class AddShowStatisticsToOrganization < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_organizations, :show_statistics, :boolean, default: false
```

Comment:
@oriolgual don't quite like setting default values on the database. Looks like something that should be done at the `model`, as well as in the `Form`. When creating new models, Rails isn't able to inspect the table for default values, so the checkbox will show unchecked.